src/RequestBuilder.php

```diff
@@ -109,7 +109,7 @@ public function build($path, Message $message, array $headers = [])
 
                 list($body, $queryParams) = $this->constructBodyAndQueryParameters($message, $config);
                 
-                // Request enum fields be encoded as numbers rather than strings.
+                // Request enum fields will be encoded as numbers rather than strings.
```

Comment:
nit: If you're describing what this query param does, the previous wording was correct.
`Request enum fields be encoded as numbers rather than strings (in the response)`

If you're saying that, additionally, the request itself will have enums as numbers, this wording is fine, but it's probably misleading to say that here, since it's not related to the effects of the query param.

Reply:
Added the `(in the response)` clarification.